GREEDY/View/MainScreen.cs

```diff
@@ -2,50 +2,72 @@
 using System.Windows.Forms;
 using GREEDY.Services;
 using GREEDY.DataManagers;
+using System.Linq;
 
 namespace GREEDY.View
 {
     public partial class MainScreen : Form
     {
         private readonly IReceiptService _receiptService;
+        private readonly IItemService _itemService;
         private readonly IImageGetter _photoImageGetter;
         private readonly IImageGetter _fileImageGetter;
 
-        public MainScreen(ReceiptService receiptService)
+        public MainScreen(IReceiptService receiptService, IItemService itemService)
         {
             _receiptService = receiptService;
+            _itemService = itemService;
             _photoImageGetter = new PhotoImageGetter();
             _fileImageGetter = new FileImageGetter();
             InitializeComponent();
         }
 
         private void InserFile_Button_Click(object sender, EventArgs e)
         {
-            var image = _fileImageGetter.GetImage();
             Application.UseWaitCursor = true;
             InserFile_Button.Enabled = false;
+            var image = _fileImageGetter.GetImage();
             var processedReceipt = _receiptService.ProcessReceiptImage(image);
-            ItemList.DataSource = processedReceipt;
+            if (processedReceipt != null)
+            {
+                ItemList.DataSource = processedReceipt;
+                ItemList.Columns[0].ReadOnly = true;
+                ItemList.Columns[1].ReadOnly = true;
+            }
             Application.UseWaitCursor = false;
             InserFile_Button.Enabled = true;
         }
 
+        private void itemList_CellValueChanged(object sender, DataGridViewCellEventArgs e)
+        {
+            if (ItemList.SelectedCells[0].Value != null)
+            {
+                _itemService.AddChangeCategory(
+                    ItemList.CurrentRow.Cells[0].Value.ToString(), 
+                    ItemList.SelectedCells[0].Value.ToString()
+                );
+            }
+        }
+
```

Comment:
Can we avoid coding in main windows form screen? Maybe, we can create a static class and do not repeat this code (because everything will be needed in Camera_Button too.